runtime/compiler/env/VMJ9Server.cpp

```diff
@@ -309,10 +309,13 @@ TR_J9ServerVM::getClassClassPointer(TR_OpaqueClassBlock *objectClassPointer)
    }
 
 void *
-TR_J9ServerVM::getClassLoader(TR_OpaqueClassBlock * classPointer)
+TR_J9ServerVM::getClassLoader(TR_OpaqueClassBlock *clazz)
    {
-   TR_ASSERT(false, "The server vm should not call getClassLoader.");
-   return NULL;
+   void *loader = NULL;
+   JITServer::ServerStream *stream = _compInfoPT->getMethodBeingCompiled()->_stream;
+   JITServerHelpers::getAndCacheRAMClassInfo((J9Class *)clazz, _compInfoPT->getClientData(), stream,
```

Comment:
Looks good. The cast to `(void *)&loader)` is not needed, though.

Reply:
Fixed. I copied this from a similar method nearby.